core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/stringfunction/RepeatExpression.kt

```diff
@@ -4,15 +4,20 @@ import ch.ergon.dope.resolvable.expression.TypeExpression
 import ch.ergon.dope.resolvable.expression.unaliased.type.toNumberType
 import ch.ergon.dope.resolvable.expression.unaliased.type.toStringType
 import ch.ergon.dope.resolvable.operator.FunctionOperator
+import ch.ergon.dope.validtype.NumberType
 import ch.ergon.dope.validtype.StringType
 
 class RepeatExpression(
     private val inStr: TypeExpression<StringType>,
-    private val n: Int,
+    private val n: TypeExpression<NumberType>,
```

Comment:
Can we have a better parameter name than `n`?

Reply:
renamed to repeatAmount